README.md

```diff
@@ -505,7 +505,7 @@ Supported platforms: any having Awesome and `curl` installed.
 * Argument: the ICAO station code, e.g. `"LDRI"`
 * Returns a table with string keys: `${city}`, `${wind}`, `${windmph}`,
   `${windkmh}`, `${sky}`, `${weather}`, `${tempf}`, `${tempc}`, `${humid}`,
-  `${dewf}`, `${dewc}` and `${press}`
+  `${dewf}`, `${dewc}` and `${press}`, `${when}`
```

Comment:
Whilst we appreciate to get extra information, IMHO it's quite cumbersome to use it. Do you think it'd be more convenient to receive the relative offset, or at least local time? As this would involve adding `get_timezone_offset` to `weather_all`, I'd like to add that `os.date` by default use current time (i.e. the one returned by `os.time`).

Reply:
Sure!  Honestly I was a little concerned that doing all the offset lookup etc when the user isn't using this field would be a bunch of wasted work.  If you don't mind always translating it to local time that's fine with me.

I still think surfacing it as this data type is best since then the user can surface it as a duration (`when - os.time()`) or some other format (`os.date`) if they want to without too much hassle

Reply:
(Done)